lib/transaction_search.rb

```diff
@@ -41,7 +41,10 @@ def define_search_method(new_method_name, old_method_name)
         do_search(@search_fields)
         add_optimizations
         sort_and_paginate
-        render :layout => @layout if @layout
+        respond_with(@order_details) do |format|
+          format.html { render layout: @layout if @layout }
+          format.csv { render text: Reports::AccountTransactionsReport.new(@order_details, date_range_field: @date_range_field).to_csv }
```

Comment:
It looks like these headers are to get IE8 to download csv files. I'm kind of curious if they are still needed or if rails includes this now. I'll do a test with IE8.